deepspeed/profiling/flops_profiler/examples/bert.py

```diff
@@ -0,0 +1,43 @@
+from functools import partial
```

Comment:
I wonder if we should move these examples to DSE? or maybe still in the DeepSpeed repo but not in the deepspeed package? I am not sure we want them to be installed with the library itself (e.g., hidden away as a library like 
 /usr/local/lib/python3.6/dist-packages/deepspeed) where they are hard to find and run directly.

Reply:
I moved the examples to readme and tests/unit/test_flops_profiler.py